### PR TITLE
Set 0 instead of null for empty block tx fields

### PR DESCRIPTION
**Pull request by codchen**
## Describe your changes and provide context
Fix for https://github.com/sei-protocol/sei-chain/issues/1859

## Testing performed to validate your change
unit test

### CODE REVIEW

File: evmrpc/block.go [hunk 1]
```diff
@@ -217,6 +218,11 @@ func EncodeTmBlock(
 						Input:            m.Msg.Bytes(),
 						Hash:             th,
 						TransactionIndex: (*hexutil.Uint64)(&ti),
+						GasPrice:         (*hexutil.Big)(utils.Big0),
+						Value:            (*hexutil.Big)(utils.Big0),
+						V:                (*hexutil.Big)(utils.Big0),
+						R:                (*hexutil.Big)(utils.Big0),
+						S:                (*hexutil.Big)(utils.Big0),
```
Comment (jewei1997) [inline evmrpc/block.go:228]: Do we know that the Graph will accept V,R,S values of 0?

**Review by jewei1997 — APPROVED** [review]
overall LGTM

File: evmrpc/block.go [hunk 2]
```diff
@@ -217,6 +218,11 @@ func EncodeTmBlock(
 						Input:            m.Msg.Bytes(),
 						Hash:             th,
 						TransactionIndex: (*hexutil.Uint64)(&ti),
+						GasPrice:         (*hexutil.Big)(utils.Big0),
```
Comment (jewei1997) [inline evmrpc/block.go:221]: Should we use EffectiveGasPrice here?

File: evmrpc/block.go (hunk 1 above)
Comment (codchen) [reply evmrpc/block.go:228]: good question. Need to confirm with them

**Pull request closed without merging**